### PR TITLE
Identify repeated options to provide multiple values

### PR DESCRIPTION
**Pull request by colin-young**
# Identify Options that Repeat to Provide Multiple Values
## Summary
- Identify `--fallbacksource` and `--source` as options that may be repeated
- Provide example of use of `-f`
- correct existing example for `-f`
## Details

See summary.
## Suggested Reviewers

@tdykstra @mairaw

### CODE REVIEW

File: docs/core/tools/dotnet-restore.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ proceeds to restore the tool's dependencies as specified in its project.json.
 
 `-s`, `--source` [SOURCE]
 
-Specifies a source to use during the restore operation. This overrides all of the sources specified in the NuGet.config file(s). 
+Specifies a source to use during the restore operation. This overrides all of the sources specified in the NuGet.config file(s). Multiple sources are provided by specifying this option multiple times.
```
Comment (mairaw) [inline docs/core/tools/dotnet-restore.md]: nit: I'd change this to Multiple sources can be provided...

File: docs/core/tools/dotnet-restore.md [hunk 2]
```diff
@@ -84,10 +84,14 @@ Restores dependencies and tools for the project in the current directory.
...
 Restores dependencies and tools for the `app1` project found in the given path.
 	
-`dotnet restore --f c:\packages\mypackages`
+`dotnet restore -f c:\packages\mypackages`
     
 Restores the dependencies and tools for the project in the current directory using the file path provided as the fallback source. 
+
+`dotnet restore -f c:\packages\mypackages -f c:\packages\myotherpackages`
 	
+Restores the dependencies and tools for the project in the current directory using the 2 file paths provided as the fallback sources.
```
Comment (mairaw) [inline docs/core/tools/dotnet-restore.md]: nit: please spell out the number 2 like "the two file paths"

**Comment by mairaw** [comment]
Thank you for your contribution @colin-young! This is great! I left a couple of minor comments. Can you address those?

**Comment by blackdwarf** [comment]
LGTM from the functionality aspect.

File: docs/core/tools/dotnet-restore.md (hunk 1 above)
Comment (colin-young) [inline docs/core/tools/dotnet-restore.md]: done...

File: docs/core/tools/dotnet-restore.md (hunk 2 above)
Comment (colin-young) [inline docs/core/tools/dotnet-restore.md]: ... and done.

And that's why you're a tech writer and I'm a developer :)

**Comment by colin-young** [comment]
I think that's got the requested changes in. Haven't tried updating a PR in the web interface before, but it looks like it's got the update.

File: docs/core/tools/dotnet-restore.md (hunk 2 above)
Comment (mairaw) [inline docs/core/tools/dotnet-restore.md]: I was a developer in a previous life :-)

But some of these suggestions come from Microsoft Style Guide so I just try to make sure we're following our conventions.

**Comment by mairaw** [comment]
The changes look good @colin-young! Thank you! @blackdwarf, do we also need to copy this to the CLI repo for the man pages?
Merging this.

**Pull request merged**